internal/tunnel/client/reconnecting.go

```diff
@@ -167,7 +167,9 @@ func (s *reconnectingSession) receive() {
 		// accept the next proxy connection
 		proxy, err := s.raw.Accept()
 		if err == nil {
-			go s.handleProxy(proxy)
+			if proxy != nil {
+				go s.handleProxy(proxy)
+			}
```

Comment:
Is this ever actually the case? Seems like we should never see a `nil, nil` return from `Accept`. One or the other should probably always be set.

Reply:
The `nil, nil` is for the "don't start a proxy and don't reconnect" case, which is the default for handling the `StopTunnel` message. Will just hit the `continue` below and go back to blocking on accepting more from muxado. Alternative could be an "everything is ok" error, but that's a little odd too.

Reply:
Removed!